MAUI/NavigationDrawer/Getting-Started.md

```diff
@@ -0,0 +1,332 @@
+        {
+            fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+        });
+
+        return builder.Build();
+        }      
+    }
+}     
+
+{% endhighlight %} 
+
+## Create a Simple .NET MAUI NaviagtionDrawer
+
+The .NET MAUI NaviagtionDrawer control is configured entirely in C# code or by using XAML markup. The following steps explain how to create a .NET MAUI NaviagtionDrawer (SfNaviagtionDrawer) and configure its elements:
+
+### Adding the .NET MAUI NaviagtionDrawer control
+
+Step 1: Add the NuGet to the project as discussed in the above reference section. 
+
+Step 2: Add the namespace as shown in the following code sample:
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    xmlns:navigationdrawer="clr-namespace:Syncfusion.Maui.NavigationDrawer;assembly=Syncfusion.Maui.NavigationDrawer"
+	
+{% endhighlight %}
+
+{% highlight c# %}
+
+    using Syncfusion.Maui.NavigationDrawer;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+Step 3: Set the control to content in `ContentPage.`
+
+{% tabs %}
+
+{% highlight xaml %}
+
+
+<navigationdrawer:SfNavigationDrawer x:Name="navigationDrawer">
+    <navigationdrawer:SfNavigationDrawer.ContentView>
+        <Grid/>
+    </navigationdrawer:SfNavigationDrawer.ContentView>
+</navigationdrawer:SfNavigationDrawer>
+
+
+{% endhighlight %}
+
+{% highlight c# %}
+          
+SfNavigationDrawer navigationDrawer = new SfNavigationDrawer();
+Grid grid = new Grid();
+navigationDrawer.ContentView = grid;
+this.Content = navigationDrawer; 
+
+{% endhighlight %}
+
+{% endtabs %}
+
+N> It is mandatory to set [`ContentView`]() for [`SfNavigationDrawer`]() on initializing.
+
+## Adjust Drawer Size
+
+The default position of the navigation pane is on the left, so let's change the drawer width to 250.
+
+{% tabs %}	
+{% highlight xaml %}
+
+<navigationdrawer:SfNavigationDrawer x:Name="navigationDrawer">
+    <navigationdrawer:SfNavigationDrawer.DrawerSettings>
+        <navigationdrawer:DrawerSettings DrawerWidth="250"/>
+    </navigationdrawer:SfNavigationDrawer.DrawerSettings>
+    <navigationdrawer:SfNavigationDrawer.ContentView>
+        <Grid/>
+    </navigationdrawer:SfNavigationDrawer.ContentView>
+</navigationdrawer:SfNavigationDrawer>
+	
+{% endhighlight %}
+{% highlight c# %} 
+
+using Syncfusion.Maui.NavigationDrawer;
+
+namespace NavigationSample;
+
+public partial class NavigationDrawerPage : ContentPage
+{
+	public NavigationDrawerPage()
+	{
+		InitializeComponent();
+        SfNavigationDrawer navigationDrawer = new SfNavigationDrawer();
+        Grid grid = new Grid();
+        navigationDrawer.ContentView = grid;
+        navigationDrawer.DrawerSettings = new DrawerSettings()
+        {
+            DrawerWidth = 250,
+        };
+        this.Content = navigationDrawer;
+    }
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+
+N> To change the side of the navigation pane, use the [`Position`]() property. Use the [`DrawerHeight`]() property to change the drawer height in the Top and Bottom positions.
```

Comment:
To change the side of the navigation pane, utilize the `Position` property. Adjust the drawer height in the Top and Bottom positions using the `DrawerHeight` property.

Reply:
Changed